networkx/algorithms/clique.py

```diff
@@ -578,7 +578,45 @@ def number_of_cliques(G, nodes=None, cliques=None):
 
     Returns a single or list depending on input nodes.
     Optional list of cliques can be input if already computed.
+
+    Parameters
+    ----------
+    G : NetworkX graph
+        An undirected graph.
+
+    nodes: list, optional (default=None)
+        A list of input nodes.
+
+    cliques : list, optional (default=None)
+        A list of cliques, each of which is itself a list of nodes.
+        If not specified, the list of all cliques will be computed
+        using :func:`find_cliques`.
+
+    Returns
+    -------
+    int or dict
+        If `nodes` is a single node, returns
+        the size of the number of maximal cliques `G` containing
```

Comment:
Just a minor wording suggestion:

```suggestion
        If `nodes` is a single node, returns the number of maximal cliques `G` containing
```